src/libs/ReportUtils.js

```diff
@@ -2518,7 +2518,18 @@ function getOriginalReportID(reportID, reportAction) {
 }
 
 /**
- * Return the pendingAction and the errors when we have creating a chat or a workspace room offline
+
```

Comment:
@Nodebrute This is still not resolved? I can see L2521 whitespace line. Can you please remove it?